components/application-mgt/org.wso2.carbon.identity.application.mgt/src/main/java/org/wso2/carbon/identity/application/mgt/dao/impl/ApplicationDAOImpl.java

```diff
@@ -1346,6 +1347,29 @@ private void updateClaimConfiguration(int applicationId, ClaimConfig claimConfig
             IdentityApplicationManagementUtil.closeStatement(storeRoleClaimPrepStmt);
         }
 
+        try {
```

Comment:
updateClaimConfiguration method looks too long. Better to add private methods meaningfully to break.